include/volk/volk_avx2_fma_intrinsics.h

```diff
@@ -16,6 +16,16 @@
 #define INCLUDE_VOLK_VOLK_AVX2_FMA_INTRINSICS_H_
 #include <immintrin.h>
 
+/*
+ * First order Newton-Raphson approximation of 1 / x
+ */
+static inline __m256 _mm256_rcp1_avx2_fma_ps(const __m256 x)
```

Comment:
why do you call this function `rcp1`? a more descriptive name would help others in the future.